build/secondary/third_party/nss/BUILD.gn

```diff
@@ -0,0 +1,1205 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
The targets in this file appear to be unused, and I can't find the sources they refer to. I think this file might be dead code?

Reply:
Almost certain it is. I didn't want to do cleanups in this patch since there may be platforms I am missing. This is just a pure moving of the directory.